conf/application.conf

```
@@ -27,19 +27,38 @@ play.http.errorHandler = "config.ErrorHandler"
 
 # Play Modules
 play.modules.enabled += "config.Module"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
 
 microservice {
   services {
-    contact-frontend {
+    auth {
       protocol = http
-      host = localhost
-      port = 9250
+      host     = localhost
+      port     = 8500
     }
   }
 }
 
+
+host = "http://localhost:10058"
+serviceId = "senior-accounting-officer-submission-frontend"
+
+hub-frontend {
+  host = "http://localhost:10056"
+}
+
+
+contact-frontend {
+  host = "http://localhost:9250"
```

It seems `contact-frontend.serviceId` needs to be set in order for the get help link to show up
```suggestion
  host = "http://localhost:9250"
  serviceId = ${serviceId}
```
due to
https://github.com/hmrc/play-frontend-hmrc/blob/61fa1215dbe78c81c202ccac56fc00cd4aafc43d/play-frontend-hmrc-play-30/src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/components/HmrcReportTechnicalIssue.scala.html#L24

which is defined as

https://github.com/hmrc/play-frontend-hmrc/blob/61fa1215dbe78c81c202ccac56fc00cd4aafc43d/play-frontend-hmrc-play-30/src/main/scala/uk/gov/hmrc/hmrcfrontend/config/ContactFrontendConfig.scala#L30

We'll need to apply the same fix on hub

And I think we should add a test in the view spec to test this link exists (maybe do this point as a separate PR cos we will want to come up with a pattern and apply to all repos)